### ISSUE TITLE
when drag&dropping windows on the window list, not all positions can be reached

### ISSUE THREAD

**Issue by calestyo** [linuxmint/Cinnamon#3574]
Hi.

When one moves a window on the window list via drag&drop, then sometimes[0] on cannot drop the window at all positions, since some of the other windows "stick together".

The following three screenshots triy to visualise what I mean:
![screen](https://cloud.githubusercontent.com/assets/919933/4533508/b6e67838-4d9c-11e4-9162-4d6239e416de.jpg)
(The screenshots go chronologically from up to down.)

The window that I drag here from right to left, is the one titled "FRITZ!Box 7490-Hilfe...". As I move it from right to left, I move over the middle window ("Screenshot from 20...") to just a bit left of the middle from the leftmost window ("FRITZ!Box - Iceweasel").
The problem: During the whole continuous move, there is no position in which the moved window would be dropped between "FRITZ!Box - Iceweasel" and "Screenshot from 20..." - at a certain point, both of these two jump to the other side, as if they'd stick together.

This is subtly related to #3573.

Cheers,
Chris.

[0] Sometimes it works however, and I couldn't yet find out, when it does and when not.

**Comment by RossBrunton** [comment]
For me it seems that when Cinnamon is restarted, all the windows "stick" together. With new windows in the session, they aren't "stuck" to anything and other windows can be placed to the right or left of them.

**Comment by calestyo** [comment]
starting with which version should this have been fixed?

**Comment by GLolol** [comment]
@calestyo Look for Cinnamon 2.6.x.

**Comment by calestyo** [comment]
well I'm using that right now (2.6.7) but the problem still persists

**Comment by VolMi** [comment]
I haven't seen this since I updated to 2.6. Do you have a new clue how to reproduce the issue? Does it affect all kinds of windows? Or maybe only certain kinds of windows...?
Does it matter to use a Cinnamon theme that is updated for Cinnamon 2.6?

**Comment by calestyo** [comment]
Hmm I just tried it again, and while it happened again after a while (i.e. at first, every position could be reached but later it didn't work anymore) I don't have an exact recipe to reproduce it.

I simply reordered the windows in the window list for a while, marked/unmarked some of them always-on-top, moved them without dropping and so on...

After a while there were again some windows in the list, which stuck together when I move another window (in the list) over them... just as described above.
